polygeist/tools/cgeist/Test/Verification/assume.cpp

```diff
@@ -0,0 +1,42 @@
+// RUN: cgeist -O0 %s -w -o - -S --function=* | FileCheck %s
```

Comment:
Do you also want to test lowering to LLVM dialect/IR?

Reply:
Mmmmm not sure that is to be tested here. That should be covered by `llvm` dialect tests, right?

Reply:
Yes, I assume so, good point.